common/match/pool.ts

```diff
@@ -144,7 +144,7 @@ export const pools: MatchPool[] = [
             subjects: { not: "[]" },
             screening: { success: true },
             isCodu: false,
```

Comment:
The `isCodu` flag should also removed here. In fact we never used the the CoDu-`registrationSource` for CoDu-Students since they could register for the research project even after registration (as distinguished from pupils).